R/drive_download.R

```diff
@@ -0,0 +1,72 @@
+#' Download a file from Google Drive.
+#'
+#' @template file
+#' @param out_path Character. Path for output file. Google files can
+#'   be downloaded to types specified [here](https://developers.google.com/drive/v3/web/manage-downloads).
+#' @param overwrite A logical scalar, do you want to overwrite a file that already
+#'    exists locally?
+#' @template verbose
+#'
+#' @examples
+#' \dontrun{
+#' ## Save "chickwts.csv" to the working directory as "chickwts.csv".
+#' drive_download(file = "chickwts.csv", out_path = "chickwts.csv")
+#'
+#' ## Save Google Sheet named "chickwts" to the working directory as
```

Comment:
Maybe say "export" so it's more clear how this differs from the scenario just above?

It might be nice (and good practice/check for us) to add a Google Doc / Word example.